suite-native/app/package.json

```diff
@@ -114,6 +118,7 @@
         "react-native": "0.76.1",
         "react-native-edge-to-edge": "^1.3.1",
         "react-native-gesture-handler": "^2.21.0",
+        "react-native-get-random-values": "^1.11.0",
```

Comment:
We use Expo crypto library for this

Reply:
Ok, seems like I can get rid of it, it's already polyfilled